### DISCUSSION TITLE
Release a bugfix version following 4.8.0

### DISCUSSION THREAD

**Discussion by gtoison** [spotbugs/spotbugs#2674] [General]
Hello,
Version 4.8.0 had a few problems that created many false positives or even caused the analysis to crash.
Most of these issues are now fixed, would it be possible to release a new version soon?

**Comment by iloveeclipse** [comment]
I have one pending PR here: https://github.com/spotbugs/spotbugs/pull/2673 that must be merged for 4.8.1.

Once this PR is merged, I would also vote for releasing 4.8.1 soon, for same reason like @gtoison.

**Comment by hazendaz** [comment]
Can do the release this weekend if all in agreement.  Will give another day to see any further comments.

**Comment by lvjp** [comment]
This will permit to release the blocker #2658 for French users.

**Comment by JuditKnoll** [comment]
I also support releasing 4.8.1 soon. I see no PR which must be merged before the release, and AFAIK all issues raised in connection with 4.8.0 got addressed.

**Comment by iloveeclipse** [comment]
4.8.1 is released.
